infra/blueprint-test/README.md

```diff
@@ -0,0 +1,408 @@
+# 1. Background and Overview
+
+Integration testing has been instrumental in establishing a complete and functional CI pipeline. Similar to most quality software engineering modules and applications, GCP Blueprints employ an integration testing framework that is triggered as part of the CI process as well.
+
+Apart from the necessity of including a testing framework as part of our GCP blueprints we took additional steps to ensure the following:
+
+- Easy to learn and to setup as part of the blueprints development process
+- Compatible with both Terraform and KRM based GCP blueprints
+
+Considering the above, our test framework has been developed (details in the following sections) with backward compatibility to allow for current tests to keep functioning.
+
+```
```

Comment:
nit: this does not need to be a codeblock. Lets use maybe italic to highlight any non code portions?

Reply:
Acked.